concent_api/core/views.py

```diff
@@ -358,20 +365,28 @@ def handle_receive_force_get_task_result_upload_for_requestor(decoded_message: m
     assert decoded_message.TYPE in message.registered_message_types
 
     current_time = int(datetime.datetime.now().timestamp())
-    decoded_message.timestamp                                      = current_time
-    decoded_message.file_transfer_token.timestamp                  = current_time
-    decoded_message.file_transfer_token.token_expiration_deadline  = current_time + settings.TOKEN_EXPIRATION_DEADLINE
-    decoded_message.file_transfer_token.operation                  = 'download'
+    file_transfer_token = message.concents.FileTransferToken(
+        token_expiration_deadline    = current_time + settings.TOKEN_EXPIRATION_DEADLINE,
```

Comment:
Please add an `assert` to check that timestamp <= deadline.

Reply:
:+1: